subcell_pipeline/simulation/readdy/_process_readdy_compression_simulations.py

```diff
@@ -0,0 +1,90 @@
+# %% [markdown]
+# # Process ReaDDy compression simulations
+
+# %% [markdown]
+"""
+Notebook contains steps for post processing of ReaDDy simulations in which a
+single actin fiber is compressed at different compression velocities.
+
+This notebook provides an example of processing a simulation series in which
+there are multiple conditions, each of which were run with multiple replicates.
+For an example of processing a simulation series with a single condition with
+multiple replicates, see `process_readdy_no_compression_simulations.py`.
+
+- [Define simulation conditions](#define-simulation-conditions)
+- [Parse simulation data](#parse-simulation-data)
+"""
+
+# %%
+if __name__ != "__main__":
+    raise ImportError("This module is a notebook and is not meant to be imported")
+
+# %%
+from pathlib import Path
+
+from subcell_pipeline.simulation.readdy.parser import parse_readdy_simulation_data
+
+# %% [markdown]
+"""
+## Define simulation conditions
+
+Defines the `ACTIN_COMPRESSION_VELOCITY` simulation series, which compresses a
+single 500 nm actin fiber at four different velocities (4.7, 15, 47, and 150
+μm/s) with five replicates each.
+"""
+
+# %%
+# Name of the simulation series
+series_name: str = "ACTIN_COMPRESSION_VELOCITY"
+
+# S3 bucket for input and output files
+bucket: str = "s3://readdy-working-bucket"
+
+# Number of simulation replicates
+n_replicates: int = 5
+
+# List of condition file keys for each velocity
+condition_keys: list[str] = ["0047", "0150", "0470", "1500"]
+
+# Number of timepoints
+n_timepoints = 200
+
+# Number of monomer points per fiber
+n_monomer_points = 200
+
+# Total number of steps for each condition
+total_steps: dict[str, int] = {
+    "0047": int(3.2e8),
```

Comment:
should these be calculated based on the velocity and the compression distance for consistency instead of being hard coded?